Core/StateManager.cs

```diff
@@ -1,25 +1,90 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using Core.Importers;
+using Core.TimeSeries;
 using Newtonsoft.Json;
 
 namespace Core
 {
 	public class StateManager
 	{
-		public static SortedSet<Transaction> Load()
+		private static Dictionary<string, TransactionsImporter> importers = new Dictionary<string, TransactionsImporter>
 		{
-			if (!File.Exists("state.json"))
+			["pb"] = new PrivatTransactionsImporter(),
+		};
+
+		public static void LoadState(string stateJson)
+		{
+			if (string.IsNullOrWhiteSpace(stateJson))
+			{
+				return;
+			}
+
+			State.Instance = JsonConvert.DeserializeObject<State>(stateJson);
+		}
+
+		public static string SaveToJson()
+		{
+			return JsonConvert.SerializeObject(State.Instance);
+		}
+
+		public static void LoadTransactions(IEnumerable<(string key, Stream stream)> files)
```

Comment:
Codacy found an issue: [Refactor this method to remove the nested type argument.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5836065)